src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -20,75 +24,95 @@ import {
     WIDGET_TYPE_DI_FIELD_VALUE,
 } from 'c/geConstants';
 
-const TOKENIZE_TIMEOUT = 10000; // 10 seconds
-const TOKENIZE_CARD_PAGE_NAME = 'GE_TokenizeCard';
-
 export default class geFormWidgetTokenizeCard extends LightningElement {
     @api cardHolderName;
 
     @track domain;
     @track visualforceOrigin;
+    visualforceOriginUrls;
```

Comment:
```suggestion
    _visualforceOriginUrls;
```

Reply:
Our team prefers to flag internal properties of LWC components with the "_" prefix - this is in line with the style that 
 the LWC team at Salesforce uses when developing their components.

Reply:
The var is used now in the [psElevateTokenhandler](https://github.com/SalesforceFoundation/NPSP/blob/70a0f7503bcc97841671f10542d4838c7e0e55da/src/lwc/psElevateTokenHandler/psElevateTokenHandler.js#L92). Should I stil update the var to `_visualforceOriginUrls`?

Reply:
Interesting ... should we just move this property to psElevateTokenHandler as well? I think the only use for it was in evaluated the origin of the post message, which is now being done in psElevateTokenHandler. It seems like it should be a 'private' property of that component?

Reply:
I was attempting to do that before during coding of the psElevateTokenHandler but it did not work. Let me try again. Might need to consult you or @psadie-sfdo  if I cannot make it work.

Reply:
The VF origin and origin URL vars have been moved into `psElevateTokenHandler`. Can you please review changes again?